addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -2911,6 +2911,7 @@ var AceEditor = DebouncedField.extend({
         if (this.aceEditor) {
             this.aceEditor.destroy();
         }
+        this._super.apply(this, arguments);
```

Comment:
to backport as well

Reply:
Yeah I thought about it afterwards, it's likely almost all the fixes will have to be backported to the relevant branch. Most of them are probably benign (e.g. this one probably doesn't matter much) but still.